pkg/api/v1alpha1/cluster_types.go

```diff
@@ -762,6 +766,126 @@ type KindAccessor interface {
 	ExpectedKind() string
 }
 
+// PackageConfiguration for installing EKS Anywhere curated packages.
+type PackageConfiguration struct {
+	// Disable package controller on cluster
+	Disable bool `json:"disable,omitempty"`
```

Comment:
We should probably enable by default. Only disable when the user needs to disable it.

Reply:
If this value is not set, it defaults to `false`, so it would take a user setting it to `true` to disable the package controller.